executables/collector/src/main/docker/docker-compose.yml

```diff
@@ -25,59 +25,60 @@ services:
     container_name: collector_mongo-data
     image: mongo:4.4.4
     ports:
-      - "27019:27017"
+      - "127.0.0.1:27019:27017"
```

Comment:
Every time I see this I get goose bumps ... I still think it's better to not alter the iptables of each developer who checks out this service.

I image a dev who does not know much about Docker and just follows the Readme, being in a public Wi-Fi network in a co-working space or so and, from my understanding, opens the dev's computer for attacks by opening all ports using by the containers ( "27018:27017"), also exposing the databases to the public in that network.

The only case where a dev actually needs to open the test ports to the outside world is if the dev wants to upload data to the collector from another physical device like a mobile phone instead of using an Android emulator or our phone simulator.

We can still add a section to the Readme to inform how to make the API available to the local network, if needed.